Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```
@@ -424,14 +446,22 @@ script:
       name: weight_1
     - description: Weight 2.
       name: weight_2
+    - name: property_comments
+      required: true
+      type: textArea
+    - description: Data center ID 1
+      name: dc1_id
+      required: true
+    - description: Data center ID 2
+      name: dc2_id
+      required: true	
```

```suggestion
    - name: property_comments
      type: textArea
    - description: Data center ID 1
      name: dc1_id
    - description: Data center ID 2
      name: dc2_id
```
The same also these arguments cannot be required
Because it breaks backward compatibility